packages/TelegramClient-UI.package/TCUEmojiHelper.class/class/removeOptionalCharactersFrom..st

```diff
@@ -0,0 +1,4 @@
+utility
+removeOptionalCharactersFrom: aCollection
+
+	^ aCollection removeAllSuchThat: [:code | self defaultOptionalCharacters includes: code]
```

Comment:
What are these OptionalCharacters? Why do we have to remove them?

Reply:
Emoji are encoded differently, by different devices, applications and operating systems. The ground truth though lies close to an encoding without VARIATION SELECTOR-16 characters. Its close enough for our purposes, so we decided to strip all of those characters on loading (and during parsing). Technically the files still containing the optional-characters in their name are not needed.

Reply:
Maybe add this as a comment ;)

Reply:
Am adding the comment to `defaultOptionalCharacters` now as that's the source of all confusion.